From 57ef690be66b39978d663da7e7862b9b9e21405a Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Thu, 26 Apr 2018 15:43:41 +0800
Subject: [PATCH] soc: rockchip: rockchip_dmc: add missing CONFIG_

Commit 1763310f3320 ("soc: rockchip: Add dummy
rockchip_pm_register_notify_to_dmc") make power domain driver
unable to receive dmc notify. That will make DDR freq stop or
power domain unable to idle or deidle.

Fixes: 1763310f3320 ("soc: rockchip: Add dummy rockchip_pm_register_notify_to_dmc")
Change-Id: I30970507ce56a663198c777ddf691ccfbf12611e
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 include/soc/rockchip/rockchip_dmc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/soc/rockchip/rockchip_dmc.h b/include/soc/rockchip/rockchip_dmc.h
index 96713f5d0dd5..453292f3c85a 100644
--- a/include/soc/rockchip/rockchip_dmc.h
+++ b/include/soc/rockchip/rockchip_dmc.h
@@ -15,7 +15,7 @@
 
 #include <linux/devfreq.h>
 
-#ifdef ROCKCHIP_PM_DOMAINS
+#ifdef CONFIG_ROCKCHIP_PM_DOMAINS
 int rockchip_pm_register_notify_to_dmc(struct devfreq *devfreq);
 #else
 static inline int rockchip_pm_register_notify_to_dmc(struct devfreq *devfreq)
-- 
2.35.3

